lib/gettext-setup/version.rb

```diff
@@ -0,0 +1,3 @@
+module GettextSetup
+  VERSION = '0.24'
```

Comment:
Is there a reason not to derive this from the git tags, like the gemspec did before?